src/utils.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+
+var readData = async function (path) {
+    return new Promise((resolve, reject) => fs.readFile(path, (err, data) => {
+        if (err) {
+            reject(err);
+        }
+        resolve(data);
+    }))
+        .then((data) => JSON.parse(data))
+        .then((context) => {
+            return context;
+        })
+        .catch((error) => {
+            console.log('Error!', error);
+            process.exit(1);
+        });
+}
```

Comment:
```suggestion
  return new Promise((resolve, reject) => fs.readFile(path, (err, data) => {
    if (err) {
      reject(err);
    }
    resolve(data);
  }))
    .then((data) => JSON.parse(data))
    .then((context) => {
      return context;
     })
     .catch((error) => {
       console.log('Error!', error);
       process.exit(1);
     });
}
```
Take care of the lint part. We used space instead of tab and a "tab" is two spaces. You can run `npm run lint` to check this